src/@test/utilsTest/AnnualRecipeValidate.test.ts

```diff
@@ -0,0 +1,83 @@
+import { AnnualRecipeValidate } from '../../utils/AnnualRecipeValidate';
+
+// Positive tests with result < 300k
+
+test('calculateAnnualRecipeScore returns the correct score', () => {
```

Comment:
O padrão estabelecido é o `it` ao invés do test, pois geralmente utilizamos uma mensagem "it should be able to calculate annual recipe score"

Reply:
Faça no restante de todos os testes.